src/nsls2api/services/sync_service.py

```diff
@@ -192,11 +192,11 @@ async def synchronize_proposal_from_pass(proposal_id: str) -> None:
         for resource in saf.Resources:
             beamline = await beamline_service.beamline_by_pass_id(str(resource.ID))
             if beamline:
-                beamline_list.append(beamline.name)
+                saf_beamline_list.append(beamline.name)
 
         saf_list.append(
             SafetyForm(
-                saf_id=str(saf.SAF_ID), status=saf.Status, instruments=saf_beamline_list
+                saf_id=str(saf.SAF_ID), status=saf.Status, instruments=set(saf_beamline_list)
```

Comment:
@JunAishima 
Is the order of this list important to consumers?

Reply:
No, should not be.